wait-for-deployment

```diff
@@ -51,28 +52,41 @@ get_available_replicas() {
 get_deployment_jsonpath() {
   local -r jsonpath="$1"
 
-  kubectl get deployment "${deployment}" -o "jsonpath=${jsonpath}"
+  kubectl --namespace ${k8sNamespace} get deployment "${deployment}" -o "jsonpath=${jsonpath}"
```

Comment:
This line was the reason the CI complained: You need to put quotes (`"`) around the variable name holding the namespace.